src/main/groovy/util/StockMovementStatusHelper.groovy

```diff
@@ -0,0 +1,83 @@
+package util
+
+import org.pih.warehouse.api.StockMovementDirection
+import org.pih.warehouse.api.StockMovementType
+import org.pih.warehouse.auth.AuthService
+import org.pih.warehouse.core.Location
+import org.pih.warehouse.order.Order
+import org.pih.warehouse.order.OrderStatus
+import org.pih.warehouse.requisition.RequisitionStatus
+import org.pih.warehouse.shipping.Shipment
+import org.pih.warehouse.shipping.ShipmentStatusCode
+
+class StockMovementStatusHelper {
+
+    Order order
+
+    Shipment shipment
+
+    StockMovementType stockMovementType
+
+    Location origin
+
+    Location destination
+
+    RequisitionStatus defaultStatus
+
+
+    StockMovementDirection getStockMovementDirection(Location currentLocation) {
+        if (currentLocation == origin) {
+            return StockMovementDirection.OUTBOUND
+        }
+        if (currentLocation == destination || origin?.isSupplier()) {
+            return StockMovementDirection.INBOUND
+        }
+        return null
+    }
```

Comment:
Wouldn't it be better to somehow adjust the code to use the method from `StockMovement`? Or it is not possible? Maybe you can make it `static` and call it by `StockMovement.getStockMovementDirection`?
![image](https://github.com/openboxes/openboxes/assets/83239466/22b082d6-6437-4efa-b797-9fd0b25c3626)

Reply:
I think it’s better to keep it in the helper class, because otherwise I’d have to refactor the existing method in the StockMovement to receive the locations in the params etc. 
I think this helper class should be „a central place” for any type of DTOs we have (StockMovement, OutboundStockMovement, etc.)

Reply:
:+1: